src/wtc-perspective-card.js

```diff
@@ -74,8 +74,12 @@ class PerspectiveCard {
     );
     this.shine = this.element.querySelector(".perspective-card__shine");
 
+    this.cardImage = this.element.querySelector(".card__img--front");
```

Comment:
Looks like this `cardImage` isn't used anywhere. Should we remove, so as to avoid confusion?

Reply:
Good spot, removed!